src/views/Settings/General.vue

```diff
@@ -65,17 +70,23 @@ export default defineComponent({
       const grade = this.grades.indexOf(this.grade);
       return grade === -1 ? 0 : grade;
     },
+    selectedPWCState(): number {
```

Comment:
if you want to delete this function, you can by making it inline with the html so like

` :modelValue="selectedPWCState  ? 0 : 1"`

Reply:
Ended up implementing the state for this cuz it's complicated - copilot just suggested this & I went w it lol